LoRDeckCodes_Tests/UnitTest1.cs

```diff
@@ -264,7 +310,7 @@ public void BadCardCodes()
             }
 
             deck = new List<CardCodeAndCount>();
-            deck.Add(new CardCodeAndCount() { CardCode = "01XX002", Count = 1 });
+            deck.Add(new CardCodeAndCount() { CardCode = "01YY002", Count = 1 });
```

Comment:
why the X's to Y's ?

Reply:
oh oops I forgot to swap this back lol, it's cause I originally set the mount targon deck code to be XX and then this test was failing